frontend/src/metabase/admin/settings/selectors/selectors.js

```diff
@@ -361,7 +373,12 @@ export const ADMIN_SETTINGS_SECTIONS = {
           { name: t`Database Default`, value: "" },
           ...(MetabaseSettings.get("available-timezones") || []),
         ],
-        note: t`Not all databases support timezones, in which case this setting won't take effect.`,
```

Comment:
we had a few of these "note" properties, which just add something to the description on a separate line.